### DISCUSSION TITLE
Azure Monitor querying management.azure.com repeatedly

### DISCUSSION THREAD

**Discussion by 4sdk** [grafana/grafana#58692] [Data Sources]
Every time I perform a query on a log analytics workspace using the Azure Monitor data source an azure.management.com API call is called to verify the existence of the workspace. I find this redundant and confusing because when there is a problem with the Azure management API I cannot query the workspace because it tries to find out if it exits and there is no indicative error. I would like the ability to remove this verification step.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
